wagtail/images/tests/test_rich_text.py

```diff
@@ -57,10 +59,12 @@ def test_expand_db_attributes_for_editor(self):
              'alt': 'test-alt',
              'format': 'left'},
         )
-        self.assertIn(
-            '<img data-embedtype="image" data-id="1" data-format="left" '
-            'data-alt="test-alt" class="richtext-image left"', result
-        )
+        self.assertIn('<img', result)
+        self.assertIn('data-embedtype="image"', result)
+        self.assertIn('data-id="1"', result)
+        self.assertIn('data-format="left"', result)
+        self.assertIn('data-alt="test-alt"', result)
+        self.assertIn('class="richtext-image left"', result)
 
```

Comment:
We could use the [`assertTagInHTML` helper](https://github.com/wagtail/wagtail/blob/c85e4a3ff0022be405e75dbd6c5594d34d2cbb18/wagtail/tests/utils.py#L186) from wagtail.tests.utils as a neater way of doing these assertions (using `allow_extra_attrs=True` if we need to leave out `src` due to the unpredictable filename).